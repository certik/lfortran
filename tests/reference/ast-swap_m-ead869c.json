{
    "basename": "ast-swap_m-ead869c",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../examples/swap_m.f90",
    "infile_hash": "eac3a7b2dda577475410ce270c5bfbf80b0c28e7e1cba7778ed766cb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-swap_m-ead869c.stdout",
    "stdout_hash": "c953b5a9fe168eabee005fd9c32e61fb87f19a69d7bc76dbe6b611dd",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}