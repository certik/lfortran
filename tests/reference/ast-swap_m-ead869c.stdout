(TranslationUnit [(Module swap_m () [] [(ImplicitNone [] ())] [(Declaration () [(SimpleAttribute AttrPrivate)] [] ()) (Declaration () [(SimpleAttribute AttrPublic)] [(swap_t [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)])) (Template swap_t [T] [(Declaration () [(SimpleAttribute AttrPrivate)] [] ()) (Declaration () [(SimpleAttribute AttrPublic)] [(swap [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)])) (DerivedType T [] (TriviaNode [] [(EndOfLine) (EndOfLine)]) [] [] []) (Interface (InterfaceHeaderName swap) () [(InterfaceModuleProcedure [swap_] [(SimpleAttribute AttrModule)] ())])] [(Subroutine swap_ [(x) (y)] [] () () [] [] [] [(Declaration (AttrType TypeType [] T None) [(AttrIntent InOut)] [(x [] [] () None ())] ()) (Declaration (AttrType TypeType [] T None) [] [(tmp [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)]))] [(= 0 tmp x ()) (= 0 x y ()) (= 0 y tmp ())] [])])] [])])
